run/helloworld/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_hello_dockerfile]
+
+# Use the official lightweight Ruby image.
+# https://hub.docker.com/_/ruby
+FROM ruby:2.7-slim
+
+# Install production dependencies.
+WORKDIR /usr/src/app
+COPY Gemfile Gemfile.lock ./
+ENV BUNDLE_FROZEN=true
+RUN gem install bundler -v 1.17.2 && bundle install --without test
```

Comment:
Can you use bundler 2.1.4 instead? Update this line here, and then update your local bundler and delete and regenerate the `Gemfile.lock`.